icu4j/main/classes/core/src/com/ibm/icu/impl/RBBIDataWrapper.java

```diff
@@ -359,8 +378,8 @@ public static RBBIDataWrapper get(ByteBuffer bytes) throws IOException {
         }
         ICUBinary.skipBytes(bytes, This.fHeader.fRuleSource - pos);
         pos = This.fHeader.fRuleSource;
-        This.fRuleSource = ICUBinary.getString(
-                bytes, This.fHeader.fRuleSourceLen / 2, This.fHeader.fRuleSourceLen & 1);
+        This.fRuleSource = new String(
```

Comment:
Please use this constructor: `String(byte[] bytes, Charset charset)`
with `StandardCharsets.UTF_8`

https://docs.oracle.com/javase/7/docs/api/java/nio/charset/StandardCharsets.html#UTF_8
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#String(byte[],%20java.nio.charset.Charset)